src/config.ts

```diff
@@ -98,6 +100,7 @@ export const DEFAULT_QUERY_CONFIG: QueryConfig = {
   numberOrdinalProportion: .05,
 
   maxBinsList: [5, 10, 20],
+  scaleBandSizes: [0, 21, 90],
```

Comment:
0 is totally unacceptable default value for this `scale.bandSize`. 

Why on the earth would people want a band size to be zero?